.github/workflows/lints.yml

```diff
@@ -29,10 +29,8 @@ jobs:
       - name: Clone the repository
         uses: actions/checkout@v3
       - run: cargo make build-contracts
-      - name: Run Contract cargo clippy
-        run: cargo make clippy
-      - name: Run cargo clippy
-        run: cargo clippy
```

Comment:
Why is clippy removed here?

Reply:
Because the `cargo make check` flow includes this kind of check.